app/services/lists/repositories_service.rb

```diff
@@ -19,6 +19,7 @@ def fetch_records
                           .select('teams.name AS team_name')
                           .select('creators.full_name AS created_by_user')
                           .select('archivers.full_name AS archived_by_user')
+                          .select('repositories.id')
```

Comment:
We already have this

Reply:
@aignatov-bio when i opened this pr it wasn't there, that's why the code conflict.